packages/main/src/RangeSlider.hbs

```diff
@@ -25,6 +29,7 @@
 
 {{#*inline "handles"}}
 	<div class="ui5-slider-handle ui5-slider-handle--start"
+		part="handle-start"
```

Comment:
Earlier, I mean to keep `handle`, so `part="handle handle-start"` would be best. This way you can create rules for both or individually easily.

Reply:
Yes I know, however there were mixed feelings in the team about this approach. I will bring it for a discussion tomorrow again and we will come up with a final decision.